### PR TITLE
[WIP] Refactor design

### PR DESCRIPTION
**Pull request by abhij1607**
This PR consist of

- refactored Nav design
- refactored categories in homepage
- added EmptyBucket on empty cart & wishlist state

### CODE REVIEW

File: src/pages/Home/categories.js [hunk 1]
```diff
@@ -1,80 +1,54 @@
...
+  useEffect(() => {
+    (async () => {
+      try {
+        const categoriesResponse = await axios.get("/api/categories");
+        dataDispatch({
+          type: "CATEGORIES",
+          payload: categoriesResponse.data.categories[0].platform,
+        });
+      } catch (error) {
+        console.log(error);
```
Comment (AnuragSahu11) [inline src/pages/Home/categories.js:18]: remove console.log

**Review by AnuragSahu11 — APPROVED** [review]
Good use of Async iffy, ready to merge

File: src/components/NavBar/nav-components/nav-categories.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ const NavCategories = () => {
       }
     })();
   }, []);
-
+  console.log(dataState.platform);
```
Comment (prachi-sahani) [inline src/components/NavBar/nav-components/nav-categories.js]: can be removed

**Review by prachi-sahani — COMMENTED** [review]
All the changes look fine
Clean code
Can be merged

File: src/components/NavBar/nav-components/nav-categories.js (hunk 2 above)
Comment (abhij1607) [reply src/components/NavBar/nav-components/nav-categories.js]: thanks for pointing it

**Pull request merged**